frontend/src/components/AvatarImage/index.tsx

```diff
@@ -1,18 +1,32 @@
 import React from 'react';
 
-import emptyUser from '../../assets/img/emptyUser.jpg';
 import './AvatarImage.scss';
 
 type Size = 'small' | 'large';
 
 interface AvatarImageProps {
   userName: string;
-  source?: string;
   size: Size;
 }
 
-const AvatarImage = ({ userName, source, size }: AvatarImageProps) => (
-  <img src={source || emptyUser} alt={`${userName}`} className={`profile-image ${size}`} />
-);
+const getInitials = (userName: string) => {
+  const nameParts = userName.split(' ');
```

Comment:
When I first logged, `userName` showed an empty `User` object, with only one prop `isLoggedIn` and the side panel on the right displayed `u u`.